### ISSUE TITLE
Timeout in loader is to short

### ISSUE THREAD

**Issue by mistoll** [robotics-in-concert/rocon_tools#113]
In rocon_interactions/src/rocon_interactions/loader.py, Line 65 there's a hard coded timeout of 15seconds. On slow systems this timeout does not allow the node to start.

Simple solution: 
- raise timeout

Better solution:
- catch exception
- log a warning
- start over (wait again)

BTW: This is a common problem, which occurs not only here, but also in other rocon packages.

**Comment by stonier** [comment]
Yes, very aware of it - we've been moving to other mechanisms in the development branches.
